docs/adr/adr_statviz_fe_sharing.md

```diff
@@ -0,0 +1,148 @@
+2. There should be options for filtering and grouping of the data in this view either globally or per visualization.
+3. We need to create a component which triggers the query and download.
+4. We need to create a function transforming the data from the graphQL query into csv format.
+
+#### What (minimal) work needs to be done on the BE side?
+
+1. We need to make the queries accessible through the private Boxtribute graphQL endpoint.
+
+#### comments
+
+Atm the filtering and grouping is happening on the FE side. The BE just returns all available data in the smallest granularity. There is the posibility to move filtering and grouping to the BE by making filters and grouping part of the graphQL query. This might make sense if we only allow the csv export.
+
+### B. SVG
+
+#### What (minimal) work needs to be done on the FE side?
+
+1. Steps 1, 2, 3 from CSV.
+2. There should be an input at least for the aspect ratio (or width and height) of the exported svg. This input could also just be given the window size of the current view. All other properties like axis label font size, axis thickness,... could be calculated based on this input. However, I can imagine that they might want more styling options that they can directly control. At the same time they could easily change the style in the exported svg through Adobe or any coding IDE.
+3. There needs to be a function transforming the data from the graphQL query into the svg. Since an svg can be easily included in the DOM, it makes sense to combine this with a preview view that is directly shown in the fancy graphs view even before the download.
+
+#### What (minimal) work needs to be done on the BE side?
+
+1. same steps as for CSV
+
+#### comments
+
+- Atm the filtering and grouping is happening on the FE side. Combined with svg previews that is a cool thing. The user can filter and group dynamically without sending requests to the BE. This means that it goes much faster for the user to find the right filter/grouping options that you want for the export of the svg.
+
+- When importing the svg into an email campaign, the svg can be responsive (through CSS) and even basic tooltips and animations (svg SMIL features) should be possible. However, not all email clients support this and probably a fallback bitmap image should be added. At the same time, I'm not sure how well the email editor in mailchimp works and probably you need to add the svg through the code editor of mailchimp.
+
+### C. bitmap (aka PNG)
+
+#### What (minimal) work needs to be done on the FE side?
+
+1. same steps as for svg.
+2. There should be a function to convert the preview svg that is in the fancy graphs view to a png.
+3. (Maybe some some more styling controls like resolution)
+
+#### What (minimal) work needs to be done on the BE side?
+
+1. same steps as for CSV
+
+#### comments
+
+- I prefer png to jpeg since the background can be transparent if they want it.
```

Comment:
jpeg has far better compression though.
this is especially a good option for organizations who want the visualization for their website and don't want to consider things like file sizes, formats or compression.

Since we use a library for the conversion of formats i hope we can offer both options as well as more modern formats like AVIF and WebP.